Libplanet.RocksDBStore/RocksDBKeyValueStore.cs

```diff
@@ -145,5 +150,10 @@ public IEnumerable<KeyBytes> ListKeys()
                 yield return new KeyBytes(it.Key());
             }
         }
+
+        public void TryCatchUpWithPrimary()
+        {
+            _keyValueDb.TryCatchUpWithPrimary();
+        }
```

Comment:
Is it needed for `RocksDBStore` case too?

Reply:
Although it might not be strictly needed, I also think having this option for `RocksDBStore` might be a good idea. 🤔

Reply:
Problem is lot more complicated than it seems. `RocksDBStore` handles many different `RocksDb` instances internally. It isn't obvious at all how catch-up should work. On the other hand, other APIs such as `StoreLoaderAttribute` clashes with having separate modes open for `RocksDBStore` and `RocksDBStateStore`. 🙄